pom.xml

```diff
@@ -64,17 +64,22 @@
     <properties>
         <revision>2.14</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.121.3</jenkins.version>
         <java.level>8</java.level>
         <workflow-cps-plugin.version>2.69</workflow-cps-plugin.version>
-        <git-plugin.version>3.6.4</git-plugin.version>
-        <scm-api-plugin.version>2.2.7</scm-api-plugin.version>
-        <workflow-scm-step-plugin.version>2.4</workflow-scm-step-plugin.version>
-        <workflow-multibranch-plugin.version>2.10</workflow-multibranch-plugin.version>
+        <git-plugin.version>3.9.3</git-plugin.version>
```

Comment:
Note: will be some merge conflicts with #65.

Reply:
Needs to be resolved.

Reply:
Done in aadda23.